### PR TITLE
[Wrangler] Add docs for the delete command

### LINKED ISSUE #6791: [Wrangler] Document `wrangler delete`
### Which Cloudflare product(s) does this pertain to?

Workers

### Subject Matter

wrangler delete

### Content Location

https://developers.cloudflare.com/workers/wrangler/commands/

### Additional information

I'll do it

### PR DESCRIPTION
**Pull request by rozenmd**
CC: @threepointone 

Closes #6791

### CODE REVIEW

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -261,6 +262,27 @@ None of the options for this command are required. Also, many can be set in your
...
+{{<definitions>}}
+
+- `SCRIPT` {{<type>}}string{{</type>}}
+  - The path to an entry point for your Worker.
+- `--name` {{<type>}}string{{</type>}}
+  - Name of the Worker.
+- `--env` {{<type>}}string{{</type>}}
+  - Perform on a specific environment.
+- `--dry-run` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: false){{</prop-meta>}}
+  - Don't actually delete the Worker. This is useful for testing the output of `wrangler delete`.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]:
```suggestion
  - Do not actually delete the Worker. This is useful for testing the output of `wrangler delete`. 
```

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -261,6 +262,27 @@ None of the options for this command are required. Also, many can be set in your
 
 ---
 
+## delete
+
+Delete your Worker, its bindings, and any associated Workers Sites or asset KV namespaces from Cloudflare.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md]: Are DOs included in the list of deleted things? If so maybe we should change wording to 'all associated Cloudflare developer platform resources'? I am not sure how much gets deleted with the Worker. But if this is all, then good as is.

**Review by deadlypants1973 — APPROVED** [review]

File: content/workers/wrangler/commands.md (hunk 2 above)
Comment (rozenmd) [reply content/workers/wrangler/commands.md]: Good catch

Comment (rozenmd) [inline content/workers/wrangler/commands.md]:
```suggestion
Delete your Worker, and all associated Cloudflare developer platform resources.
```

File: content/workers/wrangler/commands.md [hunk 3]
```diff
@@ -261,6 +262,27 @@ None of the options for this command are required. Also, many can be set in your
 
 ---
 
+## delete
+
+Delete your Worker, and all associated Cloudflare developer platform resources.
```
Comment (rozenmd) [inline content/workers/wrangler/commands.md]:
```suggestion
Delete your Worker and all associated Cloudflare developer platform resources.
```

**Pull request merged**